tests/system/test_system.py

```diff
@@ -55,6 +55,7 @@
 CREATE_INSTANCE = os.getenv("GOOGLE_CLOUD_TESTS_CREATE_SPANNER_INSTANCE") is not None
 USE_EMULATOR = os.getenv("SPANNER_EMULATOR_HOST") is not None
 SKIP_BACKUP_TESTS = os.getenv("SKIP_BACKUP_TESTS") is not None
+SPANNER_RPC_TIMEOUT_IN_SECONDS = int(os.getenv("SPANNER_RPC_TIMEOUT_IN_SECONDS", 30))
```

Comment:
This timeout is not an RPC timeout. It defines how long to poll for a result on the operation object. I would rename this `SPANNER_OPERATION_TIMEOUT_IN_SECONDS` to indicate this.

Reply:
Now renamed the env var to `SPANNER_OPERATION_TIMEOUT_IN_SECONDS`, and default it to 60 seconds.